src/util.ts

```diff
@@ -15,9 +15,14 @@ export function removeTrailingSlash(path: string) {
 
 /** Remove the subpage segment of a URL string */
 export function removeSubpageSegment(path: string) {
-	// Include new pages with subpages as part of this if statement.
-	if (/(?:install|deploy|integrations-guide|tutorial)\//.test(path)) {
-		return path.slice(0, path.lastIndexOf('/'));
+	// Include new pages with subpages as part of this regex.
+	const regex = /(?:install|deploy|integrations-guide|tutorial|migrate-to-astro|cms)\//;
+
+	if (regex.test(path)) {
+		const matchIndex = regex.exec(path)!.index;
```

Comment:
We can probably avoid the non-null assertion and running the regex twice like this, I think? `regex.exec()` returns `null` if it doesn’t match.

```suggestion
	const matches = regex.exec(path);
	if (matches) {
		const matchIndex = matches.index;
```